src/lib/tls/asio/asio_stream.h

```diff
@@ -191,10 +191,9 @@ class Stream final : public StreamBase<Channel>
 
          boost::asio::async_completion<HandshakeHandler, void(boost::system::error_code)> init(handler);
```

Comment:
just a reminder that `async_completion` is the "old way", Boost 1.70 (and Asio master/develop branches) use `async_initiate`. The old method still works, but is not quite as nice with coroutines.